desktop-ui.json

```diff
@@ -38,14 +38,15 @@
 			"button": "参与调查",
 			"versions": "6.0.0-beta6"
 		},
-		"en-IR": {
+		"fa-IR": {
```

Comment:
So one of the annoying things about the way that we're using JSON here is that javascript objects can't have duplicate keys. There's a `"fa-IR"` key on line 96 which can't be replicated.
Also - it looks like that campaign is enabled, so we don't want to just replace it.
Aaaaaand here is where our current solution gets messy because I chose not to do the more complicated refactor :/